src/main/resources/application.conf

```diff
@@ -12,4 +12,11 @@ db {
     url = ${DB_URL}
     user = ${DB_USER}
     password = ${DB_PASSWORD}
+}
+
+mail {
```

Comment:
Wir sollten diese Daten auch in eine .env-Datei packen und hier reininjecten. So wie bei der Datenbank